lib/themes/dosomething/paraneue_dosomething/js/components/Button.js

```diff
@@ -0,0 +1,37 @@
+const React = require('react');
+
+/**
+ * Button Component
+ * <Button />
+ */
+const Button = (props) => {
+    const classes = ['button'].concat(props.classes);
```

Comment:
And then the `.-reactive` class can be applied based on a prop, and stays as part of the Button's implementation.